src/frontend/src/lib/styles/global/layout.scss

```diff
@@ -1,13 +1,9 @@
 @use '../mixins/media';
 
-main,
-.main {
+header {
 	margin: 0 auto;
-	padding-inline: var(--padding-2_5x);
-
-	max-width: media.$breakpoint-small;
 }
 
-header {
-	margin: 0 auto;
+.central-content {
```

Comment:
What's central? It's the main paine?

Reply:
correct, i will change the name: `main-content`?

Reply:
Changed to `main-content`